src/PrestaShopBundle/Resources/views/Admin/Components/search_form.html.twig

```diff
@@ -0,0 +1,62 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+<form id="header_search"
+      class="bo_search_form dropdown-form js-dropdown-form collapsed"
+      method="post"
+      action="{{ baseAdminUrl }}index.php?controller=AdminSearch&amp;token={{ getAdminToken('AdminSearch') }}"
+      role="search">
+  <input type="hidden" name="bo_search_type" id="bo_search_type" class="js-search-type" />
+  {% if show_clear_btn %}
+    <a href="#" class="clear_search hide"><i class="icon-remove"></i></a>
+  {% endif %}
+  <div class="input-group">
+    <input type="text" class="form-control js-form-search" id="bo_query" name="bo_query" placeholder="{{ 'Search (e.g.: product reference, customer name…)'|trans({}, 'Admin.Navigation.Header') }}">
+    <div class="input-group-append">
+      <button type="button" class="btn btn-outline-secondary dropdown-toggle js-dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        {{ 'Everywhere'|trans({}, 'Admin.Navigation.Header') }}
+      </button>
+      <div class="dropdown-menu js-items-list">
+        <a class="dropdown-item" data-item="{{ 'Everywhere'|trans({}, 'Everywhere') }}" href="#" data-value="0" data-placeholder="{{ 'What are you looking for?'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-search"><i class="material-icons">search</i> {{ 'Everywhere'|trans({}, 'Admin.Navigation.Header') }}</a>
```

Comment:
I think "Everywhere" should be located in _Admin.Navigation.Header_

Have you created this new domain ("Everywhere") on purpose?